README.md

```diff
@@ -230,8 +230,8 @@ The input `obj` contains two properties passed in the following format:
 
 The transformation function may alter both the key and the value.
 
-The function may return either an object in the asme format as the input or a value that evaluates to false.
-If the return value is falsey, the entry will be dropped from the store, otherwise it will replace the original key/value.
+The function may return either an object in the same format as the input or a value that evaluates to false.
+If the return value is false, the entry will be dropped from the store, otherwise it will replace the original key/value.
```

Comment:
`falsey` is the correct word here, b/c it doesn't have to be purely `false`. It could be `null, undefined, 0`, etc.